.jenkins/pytorch/common_utils.sh

```diff
@@ -100,6 +100,6 @@ function checkout_install_torchvision() {
 
 function clone_pytorch_xla() {
   if [[ ! -d ./xla ]]; then
-    git clone --recursive https://github.com/pytorch/xla.git
+    git clone --recursive -b tf-update --single-branch https://github.com/pytorch/xla.git
```

Comment:
Pinned to test if this chagne would break the upstream; this will be merged into ptxla 1.11.

Reply:
Do we need to update this to point back to master or are we okay with it being pinned like this in master?

Reply:
Yes, once all CI runs turn green, I will remove the pin. Thank you @seemethere !